host/sgx/switchless.c

```diff
@@ -78,20 +84,13 @@ static void* _switchless_ecall_worker(void* arg)
 {
     oe_enclave_worker_context_t* context = (oe_enclave_worker_context_t*)arg;
 
-    // Loop until stop has been requested.
-    while (!context->is_stopping)
+    // Enter enclave to process ecall messages.
+    if (oe_sgx_switchless_enclave_worker_thread_ecall(
```

Comment:
If I understand correctly, the change here is because the while loop is also done in the `oe_sgx_switchless_enclave_worker_thread_ecall`. Is that correct?

Reply:
Yes.